packages/lib/survey/service.ts

```diff
@@ -70,10 +66,15 @@ export const selectSurvey = {
   },
 };
 
+const getActionClassIdFromTrigger = (actionClasses: TActionClass[], trigger: string): string => {
```

Comment:
please rename to `getActionClassIdFromName`
I think our naming of the triggers wasn't the best. But like this it's clearer what the function does.
also second attribute `trigger` you can accept `actionClassName` to make this even clearer